packages/front-end/components/GeneralSettings/FeaturesSettings.tsx

```diff
@@ -112,20 +128,74 @@ export default function FeaturesSettings() {
         </div>
         {hasRequireApprovals && (
           <>
-            <div className="mt-4">
-              <label className="mr-1" htmlFor="toggle-require-reviews">
-                Require approval to publish changes:
-              </label>
-            </div>
-            <div>
-              <Toggle
-                id={"toggle-require-reviews"}
-                value={!!form.watch("requireReviews")}
-                setValue={(value) => {
-                  form.setValue("requireReviews", value);
-                }}
-              />
-            </div>
+            <div className="d-inline-block h4 mt-5 mb-2">Approval Flow</div>
+            {form.watch("requireReviews")?.map?.((requireReviews, i) => (
+              <div className="appbox py-2 px-3" key={`approval-flow-${i}`}>
+                <label
+                  className="mr-1 mt-3 d-block"
+                  htmlFor="toggle-require-reviews"
```

Comment:
This is inside a loop, so the id here needs to include `i` or else it won't be unique.  Same for all the other labels in this section